### PR TITLE
Feat/ministry name

### PR DESCRIPTION
**Pull request by DerekRoberts**

### CODE REVIEW

**Comment by DerekRoberts** [comment]
@basilv How do I handle the migrations?  Like this, new migrations, just ignore for now?

File: public/src/main.ts [hunk 1]
```diff
@@ -8,3 +8,5 @@ if (environment.production) {
 }
 
 platformBrowserDynamic().bootstrapModule(AppModule);
+
+// Junk commit
```
Comment (basilv) [inline public/src/main.ts]: Revert the junk commit?

File: api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js [hunk 2]
```diff
@@ -576,7 +576,7 @@ module.exports = class forestClientTypesNonCNonI1639180924469 {
         ('00012752', 'REGIONAL DISTRICT OF EAST KOOTENAY', CURRENT_USER),
         ('00012779', 'REID, COLLINS AND ASSOCIATES LTD.', CURRENT_USER),
         ('00012786', 'HORTH FORESTRY LIMITED', CURRENT_USER),
-        ('00012797', 'MINISTRY OF FORESTS, LANDS & NATURAL RESOURCE OPERATIONS', CURRENT_USER),
+        ('00012797', 'MINISTRY OF LAND, WATER and RESOURCE STEWARDSHIP', CURRENT_USER),
```
Comment (basilv) [inline api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js]: I wasn't expecting these to change, technically we pull this data from the Client table from "THE" Oracle database, and that should be the source of truth. However, I see no reason not to make this change. (In other words, this is good.) But we should probably let the team maintaining the source data know that we expect the ministry name change to be applied.

Comment (basilv) [reply api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js]: However, these file changes won't actually change the data in existing FOM databases (e.g. in test, prod, and demo environments). To do that, you need to create a new migration that updates the existing rows needing to be changed.

File: public/src/app/comment-modal/comment-modal.component.html [hunk 3]
```diff
@@ -19,8 +19,8 @@ <h4 class="modal-title">Submit a Comment</h4>
...
-    <p>If you have any questions about the collection, use and disclosure of the information collected, contact the Manager, Policy and Land Operations Support at the Ministry of Forests, Lands, Natural Resource Operations and Rural Development.</p>
+    <p>Personal information is collected under the authority section 26(c) of the Freedom of Information and Protection of Privacy Act. Personal information includes but is not limited to your opinion, name, phone number, email and mailing address. The personal information collected is directly related to the processing of Crown land tenure applications and may be considered by the Ministry of Land, Water and Resource Stewardship further to the administration of the Land Act.</p>
+    <p>If you have any questions about the collection, use and disclosure of the information collected, contact the Manager, Policy and Land Operations Support at the Ministry of Land, Water and Resource Stewardship.</p>
```
Comment (basilv) [inline public/src/app/comment-modal/comment-modal.component.html]: Issue #46 requests the ministry be renamed to "Ministry of Forests". Maybe there's discussion I missed on using LWRS instead, but I just wanted to verify.

**Review by basilv — DISMISSED** [review]

File: api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js (hunk 2 above)
Comment (basilv) [reply api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js]: Just now saw your original question. Strictly speaking, Julius has only asked for a change to the UI screens, and I don't see a situation in which these forest client records for the ministry will actually be used, so we could most likely just ignore the existing forest client data.

File: public/src/main.ts (hunk 1 above)
Comment (DerekRoberts) [reply public/src/main.ts]: Yup!  That's just part of testing.

File: public/src/app/comment-modal/comment-modal.component.html (hunk 3 above)
Comment (DerekRoberts) [reply public/src/app/comment-modal/comment-modal.component.html]: Crap, you're right!  Fixed.

File: api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js (hunk 2 above)
Comment (DerekRoberts) [reply api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js]: How about popping into Postgres and making the correction manually?  Bad idea?

No idea about the source of truth team.  Do we have a contact?

Comment (basilv) [reply api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js]: I'm shocked as a devops person you'd suggest manually :). (Yes, its a bad idea.)
Anne-Marie knows who to contact, I believe.

Comment (DerekRoberts) [reply api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js]: Ouch.  :D  ...Thanks, I'll check in with Anne-Marie.

Comment (DerekRoberts) [reply api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js]: @basilv You're right.  The migration changes have been dropped.  Simpler now!  Review again please?

**Pull request merged**